examples/functions/trip/TR_ParallelService_OeBB_right_with_remain_in_vehicle.xml

```diff
@@ -0,0 +1,250 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<OJP xmlns="http://www.vdv.de/ojp" xmlns:siri="http://www.siri.org.uk/siri" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.1-dev" xsi:schemaLocation="http://www.vdv.de/ojp ../../../OJP.xsd">
+	<!--
+We have a parallel train NJ 490 and NJ 40490
+Changement of NJ 490 to NJ590 at the border to Germany. This should be modeled with ParallelService
+
+All the rest of the information is minimalized and is not necessary accurate.
+-->
+	<OJPResponse>
+		<siri:ServiceDelivery>
+			<siri:ResponseTimestamp>2023-08-15T20:09:36Z</siri:ResponseTimestamp>
+			<siri:ProducerRef>EFAController10.5.23.52-OJP-EFA01-P</siri:ProducerRef>
+			<siri:Status>true</siri:Status>
+			<OJPTripDelivery>
+				<siri:ResponseTimestamp>2023-08-15T20:09:35Z</siri:ResponseTimestamp>
+				<siri:Status>true</siri:Status>
+				<CalcTime>762</CalcTime>
+				<TripResult>
+					<Id>ID-48D1A6A7-2659-4433-9795-F0A0840A4938</Id>
+					<Trip>
+						<Id>ID-48D1A6A7-2659-4433-9795-F0A0840A4938</Id>
+						<Duration>PT48M</Duration>
+						<StartTime>2023-08-15T18:08:00Z</StartTime>
+						<EndTime>2023-08-15T18:56:00Z</EndTime>
+						<Transfers>1</Transfers>
+						<!-- is one even when it is a remain seated -->
```

Comment:
What does "is one even when it is a remain seated" mean? Is a parallel train? Trip?

Reply:
@herlitze @normanoffel   This is a documentation question, we need to address: Is Transfers the number of TransferLegs or is it 0 in the case of a TransferType remainInVehicle?

Reply:
When I have your answer, I adapt the PR

Reply:
@herlitze  = zero transfers.

Reply:
changed